app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -2259,7 +2266,7 @@ class BrowserTabFragment :
 
         private fun showHomeBackground(favorites: List<FavoritesQuickAccessAdapter.QuickAccessFavorite>) {
             if (favorites.isEmpty()) {
-                homeBackgroundLogo.showLogo()
+                if (hideDaxBackgroundLogo) homeBackgroundLogo.hideLogo() else homeBackgroundLogo.showLogo()
```

Comment:
Could `hideDaxBackgroundLogo` be replaced by a method param? If I'm not wrong, we can decide if we need to show the logo or not based on the emitted `CtaViewState`.

(I'm just trying to remove any global state if possible)